### PR TITLE
test: skip mount2 test on CI runners

### PR DESCRIPTION
**Pull request by ivandeex**
#### What is the purpose of this change?

This PR contains a single commit spun off #5550 

Really skip `mount2` ~~and cmount~~ test on single-CPU runners to prevent sporadic lockups.

#### Was the change discussed in an issue or in the forum before?

Discussed in https://github.com/rclone/rclone/pull/5550#issuecomment-903684789 and [commit 6a9ef27b comments](https://github.com/rclone/rclone/commit/6a9ef27b0919fed17c3865a395afec97a6ed53d3#commitcomment-55235890)

cc @ncw

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by ivandeex** [comment]
@ncw :crying_cat_face:

**Comment by ivandeex** [comment]
This PR is a spin off #5550

**Comment by ncw** [comment]
I don't want to stop running the `cmount` tests otherwise we won't be testing mount at all on the CI.

`mount2` I don't care about. I'd quite like to remove it entirely as I think it is a dead end.

In practice the `cmount` tests never lock up and work pretty reliably except for occasionally on Windows which we should really fix, rather than not run the tests!

I ran my tools to get some numbers on failures on the master branch

- https://github.com/ncw/fetch-actions-logs
    - `fetch-actions-logs -conclusion failure rclone/rclone -branch "master" action-logs-$(date -I)`
- https://github.com/ncw/parse-actions-logs
    - `parse-actions-logs action-logs-2021-09-04/*.zip`

```
output/
├── github.com／rclone／rclone／cmd／cmount
│   └── TestMount／CacheMode=off／TestDirRenameFile
│       ├── windows_386／11_Run tests.txt-1117844821.txt
│       └── windows_amd64／11_Run tests.txt-1116125759.txt
├── github.com／rclone／rclone／cmd／mount2
│   ├── TestMount／CacheMode=full／TestWriteFileOverwrite
│   │   ├── go1.15／11_Run tests.txt-1001526671.txt
│   │   └── go1.15／11_Run tests.txt-974636252.txt
│   └── TestMount／CacheMode=writes／TestWriteFileOverwrite
│       ├── go1.15／11_Run tests.txt-1151618049.txt
│       └── go1.15／11_Run tests.txt-1153852995.txt
├── github.com／rclone／rclone／cmd／mountlib
│   └── TestRc／Mount
│       └── windows_386／11_Run tests.txt-1185274020.txt
├── github.com／rclone／rclone／cmd／serve／docker
│   └── TestDockerPluginMountTCP
│       ├── mac_amd64／11_Run tests.txt-1087449011.txt
│       ├── mac_amd64／11_Run tests.txt-1117856240.txt
│       └── windows_amd64／11_Run tests.txt-1116125759.txt
└── github.com／rclone／rclone／vfs／vfscache／downloaders
    └── TestDownloaders／EnsureDownloader
        └── mac_amd64／11_Run tests.txt-1001337589.txt
```

So this shows that `mount2` was responsible for 4 failures, `cmount` for 2 windows failures etc.

So in conclusion

- I support turning off testing for `mount2`
    - Possibly we could remove `mount2` entirely...
- I'd like to keep testing for `cmount` and fix the tests on Windows.

**Comment by ivandeex** [comment]
> * I support turning off testing for `mount2`

done   

> * I'd like to keep testing for `cmount` and fix the tests on Windows.

done

File: cmd/mount2/mount_test.go [hunk 1]
```diff
@@ -3,11 +3,15 @@
...
 
 import (
+	"runtime"
 	"testing"
 
 	"github.com/rclone/rclone/vfs/vfstest"
 )
 
 func TestMount(t *testing.T) {
+	if runtime.NumCPU() <= 2 {
```
Comment (ncw) [inline cmd/mount2/mount_test.go:13]: These tests are just unreliable rather than locking up - it is only `cmd/mount` which does that. So I'd suggest skipping them with `testy.SkipUnreliable(t)` instead :-)

https://github.com/rclone/rclone/blob/f7f4468cbc5cb349d6c6e4b9f70d69e023a8406a/fstest/testy/testy.go#L9-L10

**Review by ncw — COMMENTED** [review]
See inline :-)

File: cmd/mount2/mount_test.go (hunk 1 above)
Comment (ivandeex) [reply cmd/mount2/mount_test.go]: done

Comment (ivandeex) [reply cmd/mount2/mount_test.go]: I can see that we don't `export CI` in our scripts:
```
grep -r CI ./Makefile ./.github/workflows/
```

So you depend on github, right?
https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Comment (ivandeex) [reply cmd/mount2/mount_test.go]: Is this `CI` variable a standard?
Will gitlab/travis/circleci/jenkings set it to?
I guess you'd better add a link to github CI docs in `testy.go` ....

**Review by ncw — APPROVED** [review]
LGTM - thank you :-)
